modules/schema/src/main/java/org/apache/ignite/internal/schema/Column.java

```diff
@@ -30,7 +29,7 @@
  */
 public class Column {
     /** Default "default value supplier". */
-    private static final Supplier<Object> NULL_SUPPLIER = () -> null;
+    private static final DefaultValueProvider NULL_SUPPLIER = DefaultValueProvider.constantProvider(null);
```

Comment:
Could we add 'nullProvider' into DefaultValueProvider? I think it common case and we could save a memory.

Reply:
done